packages/web-components/src/components/va-on-this-page/va-on-this-page.tsx

```diff
@@ -11,12 +11,12 @@ import { Component, h } from '@stencil/core';
 })
 export class VaOnThisPage {
   render() {
-    const h2s = Array.from(document.querySelectorAll('article h2')) as Array<
-      HTMLElement
-    >;
+    const h2s = Array.from(
+      document.querySelectorAll('article h2'),
+    ) as Array<HTMLElement>;
 
     return (
-      <nav aria-label="table of contents">
+      <nav aria-labelledby="on-this-page">
```

Comment:
This looks good to me. @joshkimux can you double check that this resolves your original issue? Here's a link:

https://60f9b557105290003b387cd5-ftihhmctrh.chromatic.com/?path=/docs/components-va-on-this-page--default

Reply:
yes! This does the trick 😄